src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.NativeAot.cs

```diff
@@ -292,15 +301,24 @@ public NativeObjectWrapper(IntPtr externalComObject, ComWrappers comWrappers, ob
                 _proxyHandle = GCHandle.Alloc(comProxy, GCHandleType.Weak);
             }
 
-            ~NativeObjectWrapper()
+            public void Release()
             {
                 _comWrappers.RemoveRCWFromCache(_externalComObject);
```

Comment:
Is it possible that a NativeObjectWrapper instance could be the last object keeping the `_comWrappers` instance alive?
If not, it's possible that the `_comWrappers` field be nulled-out by the GC before this is called from the finalizer.
Do we need to add a null check here?